docs/architecture/distributed-cloud-native-apps-containers/api-gateways/gateway-patterns.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Gateway patterns
+description: Architecture for Distributed Cloud-Native Apps with .NET Aspire & Containers | Gateway patterns
+author: 
+ms.date: 04/25/2024
+---
+
+# Gateway patterns
+
+[!INCLUDE [download-alert](../includes/download-alert.md)]
+
+!TODO - sorry couldn't get these done in time. Should be easy to refresh from my powerpoint.
```

Comment:
I know these are draft make sure all Todo are taken care of

Reply:
Done. Except for the links in the download-alert.md include files, which I don't think we can complete yet.